staging/src/k8s.io/apiserver/plugin/pkg/audit/buffered/buffered_webhook_v1alpha1_test.go

```diff
@@ -29,6 +29,7 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	auditinternal "k8s.io/apiserver/pkg/apis/audit"
 	auditv1alpha1 "k8s.io/apiserver/pkg/apis/audit/v1alpha1"
+	pluginwebhook "k8s.io/apiserver/plugin/pkg/audit/webhook"
```

Comment:
Still not addressed? Just use a fake backend.

Reply:
confused

Reply:
I already extract a `pluginwebhook.NewFakeWebhookHandler`

Reply:
Also, If we can use the real webhook backend to test, why need a faked one. Our target is to test our real code, and it works , that's what we want.  I think only a fake webhook server is needed, as it is now.

Reply:
Sorry, I meant create a fake audit backend, not a fake webhook. E.g.
```go
type fakeBackend struct {
  processedEvents [][]audit.Event
  shutdown bool
}

func (f *fakeBackend) ProcessEvents(events ...*auditinternal.Event) {
  f.processedEvents = append(processedEvents, events)
}

func (f *fakeBackend) Shutdown() {
  f.shutdown = true
}

func (f *fakeBackend) Run(stopCh <-chan struct{}) error {
  retrun nil
}
```

Then this test doesn't need to deal with http test servers and what not.

Reply:
I know your meaning. It's hard to say which is better. I find many http test server ut cases in kubernetes repo. If you insist on fakeBackend, I will use fakeBackend instead of fake webserver.

Reply:
@tallclair Is this what you want ? Please take another look.